prog/prio.go

```diff
@@ -199,15 +200,19 @@ func (target *Target) BuildChoiceTable(corpus []*Prog, enabled map[*Syscall]bool
 			delete(enabled, call)
 		}
 	}
-	var enabledCalls []*Syscall
+	var generatableCalls []*Syscall
```

Comment:
It seems that the choice table is only used for call generation. Maybe just filter out the non-generatable calls and only operate on the rest? Then changes to the file would be much more compact.

Reply:
Ah, I see, the choice table was and is still in charge of keeping track of the just enabled/disabled calls.